modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceUnitsOfMeasure.testcase

```diff
@@ -608,4 +608,225 @@ definition {
 		}
 	}
 
+	@description = "This is a test for COMMERCE-12324. The admin can verify the inventory information from the SKU/UOM combination on the Inventory admin page"
+	@priority = 5
+	test VerifyUOMInventoryInfoOnInventoryPage {
```

Comment:
Why did you omit Can in the beginning? We almost always start with Can, do you have a specific reason?